### ISSUE TITLE
ffuf output style

### ISSUE THREAD

**Issue by ranjbarhadi** [ffuf/ffuf#665]
thanks for your amazing endeavors. amazing tool. here is what I'm confused at. after updating ffuf  (1.5 -> 2.0) the output is like this:
```bash
ffuf -u https://example.com/FUZZ -w ~/SecLists/Discovery/Web-Content/raft-large-directories-lowercase.txt -o kiwi-directory-all

        /'___\  /'___\           /'___\       
       /\ \__/ /\ \__/  __  __  /\ \__/       
       \ \ ,__\\ \ ,__\/\ \/\ \ \ \ ,__\      
        \ \ \_/ \ \ \_/\ \ \_\ \ \ \ \_/      
         \ \_\   \ \_\  \ \____/  \ \_\       
          \/_/    \/_/   \/___/    \/_/       

       v2.0.0-dev
________________________________________________

 :: Method           : GET
 :: URL              : https://kiwi.com/FUZZ
 :: Wordlist         : FUZZ: /root/SecLists/Discovery/Web-Content/raft-large-directories-lowercase.txt
 :: Output file      : kiwi-directory-all
 :: File format      : json
 :: Follow redirects : false
 :: Calibration      : false
 :: Timeout          : 10
 :: Threads          : 40
 :: Matcher          : Response status: 200,204,301,302,307,401,403,405,500
________________________________________________

[Status: 403, Size: 16, Words: 3, Lines: 1, Duration: 16ms]
    * FUZZ: themes

[Status: 403, Size: 16, Words: 3, Lines: 1, Duration: 18ms]
    * FUZZ: profiles

[Status: 403, Size: 16, Words: 3, Lines: 1, Duration: 18ms]
    * FUZZ: user

[Status: 403, Size: 16, Words: 3, Lines: 1, Duration: 19ms]
    * FUZZ: add

[Status: 403, Size: 16, Words: 3, Lines: 1, Duration: 20ms]
    * FUZZ: components

[Status: 403, Size: 16, Words: 3, Lines: 1, Duration: 19ms]
    * FUZZ: reply

[Status: 403, Size: 16, Words: 3, Lines: 1, Duration: 20ms]
    * FUZZ: language

[Status: 403, Size: 16, Words: 3, Lines: 1, Duration: 20ms]
    * FUZZ: contact
...
```

I like the old way which showed everything like this:
[![asciicast](https://asciinema.org/a/211350.png)](https://asciinema.org/a/211350)

before I could just look at the output and figure out everything but now it's too messy! is downgrading the only option?

**Comment by Spiff77** [comment]
Maybe i am wrong, but have you tried to add the filtering parameter (-fs) on page that actually are not changing according to parameters?
in your case: 
`ffuf -u https://example.com/FUZZ -w ~/SecLists/Discovery/Web-Content/raft-large-directories-lowercase.txt -o kiwi-directory-all -fs 16`

**Comment by sk0x1234** [comment]
check the output of the simple directory scan below,
simple chaos on the terminal. 
![image](https://user-images.githubusercontent.com/18193527/229690673-96e1193e-1863-463e-9fc5-0d4406551f70.png)

**Comment by alasalamont** [comment]
To be honest, I still feel wfuzz is the best as following reasons.
1) The output is nice looking. I also have the same issue with you
2) Does not support when fuzzing old TLS/TLS

**Comment by m57** [comment]
I agree with this, I love the tool and use it over anything else, but now the output has changed, I will be switching back to wfuzz, this is too messy.

**Comment by csandker** [comment]
+1 on this. Would love to have the old output style back (`-of oneline`, `-of grep` or something like that would be an option)

**Comment by michaelklaan** [comment]
![image](https://user-images.githubusercontent.com/40411471/233233852-74357ceb-6bc2-4dcb-abb4-6c393cd04fe4.png)

the previous output was the tool's charm, please revert to everyone's old favorite output.

**Comment by joohoi** [comment]
This was fixed in #656
